bin/steps/collectstatic

```diff
@@ -10,7 +10,9 @@
 #   - $DISABLE_COLLECTSTATIC: disables this functionality.
 #   - $DEBUG_COLLECTSTATIC: upon failure, print out environment variables.
 
-# This script is run in a subshell via sub_env so doesn't inherit the vars/utils from `bin/compile`.
+# This script is run in a subshell via sub_env so doesn't inherit the options/vars/utils from `bin/compile`.
+# TODO: Stop running this script in a subshell.
+set -eo pipefail
```

Comment:
```suggestion
set -euo pipefail
```

(right?)

Reply:
Eventually, but it breaks things if I enable it now. I was going to enable `-u` at the same time as doing to for `bin/compile`. There are just too many things to fix/change in one PR, so I'm splitting it up.